### DISCUSSION TITLE
❌ 🐞 Bug Report: Negative impact of TOC on CLS

### DISCUSSION THREAD

**Discussion by hadiomid** [elementor/elementor#21293] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The use of TOC has a negative effect on the CLS Score. Although the closed mode is selected in the settings, but at the moment of loading the page, the TOC is open and then it is closed, which is the reason for the negative effect on the CLS Score.

### Describe the solution you'd like

If the closed mode is selected for the TOC, it should not show the TOC body until the user clicks/taps on it.

**Comment by nicholaszein** [comment]
Hello @hdd064,

Thanks for your feedback! 🙏 

### ❌ 🐞 While reviewing your request we noticed you are reporting a Bug. 

#### ⚠️ Note that GitHub Discussions are reserved for 🚀 Feature Requests, and should not be used for Bug Reports.

If you believe you found a Bug and want to report it, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.


### ⚠️ As a gentle reminder, Bug Reports posted as Feature Requests may need to be closed without troubleshooting. 
Remember to always follow our **[🐞 Bug Report Guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&projects=&template=bug.yml)** to save time and ensure smooth troubleshooting.

Thank you for understanding! 🙏 

Kind regards
